src/cephadm/cephadm

```diff
@@ -1543,22 +1559,26 @@ def command_deploy():
                 raise Error("config-json parameter is needed when deploying prometheus service")
 
             uid, gid = extract_uid_gid(file_path='/etc/prometheus')
-            # Monitoring metadata is nested dicts, so asking mypy to ignore
-            p = Monitoring.components['prometheus']  # type: ignore
-            metadata = p.get('image', dict())  # type: ignore
-            monitoring_args = [
-                '--user',
-                str(uid),
-                '--cpus',
-                metadata.get('cpus', '2'),  # type: ignore
-                '--memory',
-                metadata.get('memory', '4GB')  # type: ignore
-            ]
-        else:
-            raise Error("{} not implemented in command_deploy function".format(daemon_type))
+        elif daemon_type == 'node-exporter':
+            uid, gid = 65534, 65534
```

Comment:
where does `65534` come from? Maybe adding a comment would be great.

Reply:
That comes from the user "nobody" on the distro used by the docker container for node-exporter.